### DISCUSSION TITLE
Bitbucket Login Via Github Desktop

### DISCUSSION THREAD

**Discussion by KyanR** [community/community#24062] [Apps, API and Webhooks]
<p>Not sure if I should ask here or on bitbucket forums.</p>
<p>I’m trying to log into my bitbucket account via Github desktop to access a repo I am working on. But it doesn’t accept my password. I use the same login I use for the bitbucket website and it doesn’t work.</p>
<p>It used to work but one day I logged in since then it’s been asking for my login again.</p>
<p><img src="https://user-images.githubusercontent.com/54879367/181081772-54dcd17c-b6d5-4eba-9ecc-375a72270c8b.jpeg" alt="bitbucket login" data-base62-sha1="zkPgiWcZ42XTsw3uxmQMznM8iNA" width="441" height="371"></p>

**Comment by jsoref** [comment, marked as answer]
<p>Try:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://support.atlassian.com/bitbucket-cloud/docs/create-an-app-password/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2119212/181081775-19804eda-c00b-453b-8027-ab31a6112b30.png" class="site-icon" width="32" height="32">

      <a href="https://support.atlassian.com/bitbucket-cloud/docs/create-an-app-password/" target="_blank" rel="noopener">Atlassian Support</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.atlassian.com/bitbucket-cloud/docs/create-an-app-password/" target="_blank" rel="noopener">Create an App password | Bitbucket Cloud | Atlassian Support</a></h3>

  <p>How to create a Bitbucket App password for authenticating or connecting with Bitbucket Cloud</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by KyanR** [comment]
<p>Cheers, that worked.</p>
